packages/@o3r/components/builders/metadata-check/helpers/config-metadata-comparison.helper.ts

```diff
@@ -0,0 +1,65 @@
+import { ComponentConfigOutput } from '@o3r/components';
+import { MetadataComparator } from '@o3r/extractors';
+
+/**
+ * Interface describing a config migration element
+ */
+export interface MigrationConfigData {
+  libraryName: string;
+  configName: string;
+  propertyName?: string;
+}
+
+/**
+ * Comparator used to compare one version of config metadata with another
+ */
+export class ConfigMetadataComparator implements MetadataComparator<ComponentConfigOutput> {
```

Comment:
There is no really state in this class, it sounds more like an set of helpers that a class :S